### ISSUE TITLE
long postprocessing

### ISSUE THREAD

**Issue by svigerske** [coin-or/Cbc#168]
Issue created by migration from Trac.

Original creator: @svigerske

Original creation time: 2016-06-14 13:46:48

Assignee: @tkralphs

Version: 2.9

If I run the attached instance with a timelimit of 520 seconds, the current Cbc stable/2.9 (rev 2279) takes very long in postprocessing.

That is, there are lines like
```
Cbc0003I Exiting on maximum nodes
Cbc0005I Partial search - best objective 1e+50 (best possible -38096162), took 10927 iterations and 200 nodes (343.07 seconds)
...
Cbc0005I Partial search - best objective -37399096 (best possible -38600836), took 0 iterations and 0 nodes (412.40 seconds)
```
in the output, but after printing
```
Cgl0014I Postprocessing changed objective from -37399091 to -37399098 - possible tolerance issue - try without preprocessing
```
it takes very long to terminate.

Also the final summary indicates that Cbc spend more than an hour for the postprocessing, even though it has already almost hit the timelimit before that:

```
Time (CPU seconds):             4559.80
Time (Wallclock seconds):       4568.20

Total time (CPU seconds):       4562.79   (Wallclock seconds):       4571.37
```

I attach the MPS file and a log from running with -loglevel 4.

**Comment by svigerske** [comment]
Attachment [cbc.mps.bz2](https://www.coin-or.org/issue-attachments/Cbc/ticket168/cbc.mps.bz2) by @svigerske created at 2016-06-14 13:47:59

instance to reproduce the problem

**Comment by svigerske** [comment]
Attachment [log520](https://www.coin-or.org/issue-attachments/Cbc/ticket168/log520) by @svigerske created at 2016-06-14 13:49:00

logfile

**Comment by svigerske** [comment]
Comment by @svigerske created at 2016-06-14 13:54:03

Originally, we were running this instance from GAMS with a timelimit of 3600s.
With GAMS/Cbc, I can also enable the log from Clp.
It shows that after the `Postprocessing changed objective` line, a lot of LP solves happen again, I even see some output about strong branching.

**Comment by svigerske** [comment]
Attachment [gamslog.bz2](https://www.coin-or.org/issue-attachments/Cbc/ticket168/gamslog.bz2) by @svigerske created at 2016-06-14 13:54:39

log from run under GAMS

**Comment by svigerske** [comment]
This problem seems to persist with current Cbc/master.
All that I believe would need to be done is to pass the remaining time to the postprocessing solve.

**Comment by svigerske** [comment]
I think this has been fixed by @jjhforrest via fe2bc46.
